autorag/utils/util.py

```diff
@@ -9,8 +9,11 @@
 def fetch_contents(corpus_data: pd.DataFrame, ids: List[List[str]]) -> List[List[str]]:
     assert isinstance(ids[0], list), "ids must be a list of list of ids."
     id_df = pd.DataFrame(ids, columns=[f'id_{i}' for i in range(len(ids[0]))])
-    contents_df = id_df.swifter.applymap(
-        lambda x: corpus_data.loc[lambda row: row['doc_id'] == x]['contents'].values[0])
+    try:
+        contents_df = id_df.swifter.applymap(
+            lambda x: corpus_data.loc[lambda row: row['doc_id'] == x]['contents'].values[0])
+    except IndexError:
```

Comment:
add logger.Error for logging error.
`logger.error("doc_id does not exist in corpus_data.")`